### PR TITLE
Modify createLocalDir method in Client class

### PR DESCRIPTION
**Pull request by rooneyshuman**
### Overview 
Completed modifications to the `createLocalDir()` method and its related unit test. 

Modifications to the Client class include:
* **Invoking the logger at the beginning of the method to match logging functionality included in other methods**
* **Update to the javadoc method comments**
  * New wording provides a complete yet concise description of the expected parameters and method functionality
* **Improved overall functionality and logic by removing unnecessary loop and exiting gracefully with helpful messages in case there was an issue overwriting or creating a directory.**
* **Moved prompt for the name of the directory to be created to `Main()` in order to match the `deleteRemoteFile()` method.**

### CODE REVIEW

File: src/main/java/edu/pdx/cs/sftp/Client.java [hunk 1]
```diff
@@ -529,52 +529,43 @@ void renameRemote() throws SftpException {
...
+        out.println(dirName + " has been created");
+      } catch (Exception e) {
+        out.println("Error creating directory.");
+      }
     }
-    return pass;
   }
 
-  /** Displays log history to user */
+  /** Displays log history to user and logs being invoked */
```
Comment (carissaallen) [inline src/main/java/edu/pdx/cs/sftp/Client.java:568]: What do you mean by "and logs being invoked"?

File: src/main/java/edu/pdx/cs/sftp/Client.java [hunk 2]
```diff
@@ -529,52 +529,43 @@ void renameRemote() throws SftpException {
     }
   }
 
-  /** Wrapper to create a directory on the user's local machine. */
-  void createLocalDir() {
+  /**
+   * Creates a directory in the user's current path. If a directory with the same name already
+   * exists, the user is prompted to determine whether or not to overwrite it.
+   *
+   * @param dirName the string containing the name of the directory to be created.
```
Comment (carissaallen) [inline src/main/java/edu/pdx/cs/sftp/Client.java:536]: This doc comment is **much** more readable and descriptive.

**Review by carissaallen — APPROVED** [review]
Great improvements! I didn't see a critical analysis in our document yet; will you be adding this later?

File: src/main/java/edu/pdx/cs/sftp/Client.java (hunk 1 above)
Comment (rooneyshuman) [reply src/main/java/edu/pdx/cs/sftp/Client.java:568]: it creates a log entry which says that the method has been called (see line 571)

**Comment by rooneyshuman** [comment]
> Great improvements! I didn't see a critical analysis in our document yet; will you be adding this later?

Yup. I'll add in the analysis for this PR and my previous one.

**Pull request merged**